### PR TITLE
Diff users

### PR DESCRIPTION
**Pull request by julies642**
Implemented type of user (Student or Staff) and SID attributes for User model and added corresponding fields in view forms and users page. Includes tests to create or edit a user with these attributes.

### CODE REVIEW

File: app/views/users/index.html.haml [hunk 1]
```diff
@@ -1,18 +1,24 @@
 %h1 Users
 
+= link_to 'New User', new_user_path, :class => 'btn btn-success'
```
Comment (csungwon) [inline app/views/users/index.html.haml:3]: Thank God we moved the New User button to the top. Thanks you so much

File: features/step_definitions/create_steps.rb [hunk 2]
```diff
@@ -9,11 +9,11 @@
   }
 end
 
-Given(/^I am logged in on the "([^"]*) page$/) do |arg|
-  visit "/org"
-  click_link "Log in with GitHub"
-  visit "/org"
-end
+# Given(/^I am logged in on the "([^"]*) page$/) do |arg|
```
Comment (csungwon) [inline features/step_definitions/create_steps.rb]: unnecessary comment?

File: features/step_definitions/web_steps.rb [hunk 3]
```diff
@@ -76,10 +76,17 @@ def with_scope(locator)
 # TODO: Add support for checkbox, select or option
 # based on naming conventions.
 #
+# When /^(?:|I )fill in the following:$/ do |fields|
```
Comment (csungwon) [inline features/step_definitions/web_steps.rb]: Please Please do not edit this file. Everyone is referencing to this file, and we do not want to change everyone's code. We will eventually discuss about using this file at all, but please do not change this file

**Review by csungwon — CHANGES_REQUESTED** [review]
Everything looks good to me other than features/step_definitions/webstep.rb. Please make your own step definition if needed, not changing this file.

**Review by kimhoang73 — CHANGES_REQUESTED** [review]
Please pull the latest master and rebase your branch off of it! Since your version doesn't have everyone else's feature files, merging this would delete them on master (causing Sandy and my stuff we recently merged to fail)

**Pull request closed without merging**